sql/mysql_db/privileged_database_provider.go

```diff
@@ -157,15 +157,15 @@ func (pdb PrivilegedDatabase) GetTableNames(ctx *sql.Context) ([]string, error)
 		privSet := pdb.grantTables.UserActivePrivilegeSet(ctx)
 		dbSet := privSet.Database(pdb.db.Name())
 		// If there are no usable privileges for this database then no table is accessible.
-		if privSet.Count() == 0 && !dbSet.HasPrivileges() {
+		privSetCount := privSet.Count()
```

Comment:
(Personal Preference) We can probably revert this since we're not reusing the count

Reply:
We're reusing the count on line 173

Reply:
I see, I missed the deletion on line 168 XD my bad